diagnostic.md

```diff
@@ -45,6 +57,10 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+<!-- solution below -->
+weather === "sunny" evaluates to true, because the value of "sunny" was assigned
+to the weather variable in the prior line. weather === "sunny" is a check for
+equality. Since both values are the same, the expression returns true.
```

Comment:
Good job you just forgot the first line, `let weather` evaluates to `undefined`.